handler/src/main/java/io/netty/handler/ssl/OpenSsl.java

```diff
@@ -69,7 +76,16 @@ public final class OpenSsl {
     private static final boolean IS_BORINGSSL;
     private static final Set<String> CLIENT_DEFAULT_PROTOCOLS;
     private static final Set<String> SERVER_DEFAULT_PROTOCOLS;
-    static final Set<String> SUPPORTED_PROTOCOLS_SET;
+
+    private static final int SSL_V2_HELLO  = 1;
+    private static final int SSL_V2        = 1 << 1;
+    private static final int SSL_V3        = 1 << 2;
+    private static final int TLS_V1        = 1 << 3;
+    private static final int TLS_V1_1      = 1 << 4;
+    private static final int TLS_V1_2      = 1 << 5;
+    private static final int TLS_V1_3      = 1 << 6;
+    private static int supportedProtocolsPacked = 0;
```

Comment:
make this final by using a local variable in the static initializer and setting the field only once